### ISSUE TITLE
Linux Installer

### ISSUE THREAD

**Issue by ChrisBeaumont** [glue-viz/glue#147]
Can we create a linux binary similar to the Mac binary?

**Comment by astrofrog** [comment]
I think glue should now be included in the Debian package manager: https://packages.debian.org/sid/main/glueviz

It would be nice to also get it included in ``yum`` for example, so I'll see if anyone can package that up.

Of course, users can also now use Anaconda, so the barrier is low enough that I don't think we need a standalone binary installer anymore.

@makmanalp - are you satisfied with current available options for Linux systems, or would you still want to see a standalone installer?

**Comment by makmanalp** [comment]
@astrofrog I appreciate you picking this up again.

I think there's two usage modes for glue. If you're using it more like an "app", separate from everything, I think anything goes, really, and either distro packages or binaries are fine. Bundled binaries work on a larger variety of systems and are easy for newbies, but whatever. But if, for example, you're using `qglue`, that's when the trouble starts.

I have a lot of strong feelings towards distro-specific packages - they are often messed with in unexpected ways by maintainers. Example: ubuntu's extremely frustrating insistence in breaking virtualenvs and pip for everyone and then saying it's all good: https://bugs.launchpad.net/ubuntu/+source/python3.4/+bug/1290847

Putting that trouble aside for a moment, installing things globally with an OS package manager means that you can't get to them from within a virtualenv - which is arguably the idiomatic way to store dependencies instead of polluting the global package space for every project you ever work on. The anaconda thing solves this if you do use anaconda, but that requires you to change how you do things.

It looks like pip install now actually kinda works. I like how you separated the gross dependencies. PyQT is just a huge pain. Pyside works with pip also but to get pyside you still need XCode and not just the command line tools in OSX, and I bet the linux one works fine.

Anyway, tl;dr: It'd be lovely if everything worked with a pip install, but I get that that's not simple. Maybe a separate qglue-only package would be awesome just to install that within the env, and glue outside, but I'm not sure how that'd work.

Cheers :)
